custom_components/sony/manifest.json

```diff
@@ -1,9 +1,13 @@
 {
   "domain": "sony",
   "name": "Sony Legacy API",
+  "config_flow": true,
   "documentation": "https://github.com/alexmohr/media_player.sony",
   "dependencies": [],
   "codeowners": [],
   "version": "0.1.1",
-  "requirements": ["sonyapilib==0.5.0"]
+  "requirements": ["sonyapilib==0.5.0"],
```

Comment:
With the current master, volume functions are broken because we require 0.5.0 which does not include the new functions yet.
Before accepting this release a new version in sonyapilib has to be released. This comment is as reminder to do that.

Reply:
Wouldn't be better to release the new version of sonyapilib now to fix currently broken master?

Reply:
Would be better but I don't have time to do so at the moment, so it stays broken until then.

Reply:
No problem, just saying